cotk/metric/metric.py

```diff
@@ -993,7 +993,7 @@ def close(self):
 			"fw-bw-bleu" : fw_bw_bleu \
 		})
 
-		self._hash_relevant_data(self.refs)
+		self._hash_relevant_data(self.refs + [sample_hyps, sample_refs])
```

Comment:
also add self.seed

Reply:
Add warning in docstring, if sample_hyps < sample, the hash value can be different.